### PR TITLE
New results dashboard web application

### PR DESCRIPTION
**Pull request by urig**
This PR introduces Entropy's brand new "results dashboard" web application. The results dashboard lets users explore their Entropy experiment results in graphic form and lets them compare them by overlaying plots one on top of the other.

image

To launch the new dashboard in a browser window, install Entropy and run `entropy serve <path_to_entropy_project>`.

Developers: You can launch the new dashboard from code with `python entropy-lab/cli/main.py serve <path_to_entropy_project> --debug`.

### CODE REVIEW

File: entropylab/results/dashboard/app.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Run this app with `python app.py` and
```
Comment (qguyk) [inline entropylab/results/dashboard/app.py]: @urig what is this file?

File: entropylab/results/dashboard/dashboard_data.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                        experiment_id=exp_id,
+                        id=-1,
+                        label=f"Automatic {result.label}",
+                        story=result.story,
+                        plot_data=data,
+                    )
+                )
+        return plots
+
+    def freestyle_query(self, query):
```
Comment (qguyk) [inline entropylab/results/dashboard/dashboard_data.py:50]: please delete the unused code

File: entropylab/results/dashboard/dashboard.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    _title = f"Entropy - {project_name(path)} [{project_path(path)}]"
+    app.title = _title
+
+    # Fetching data from DB
+
+    dashboard_data_reader = SqlalchemyDashboardDataReader(SqlAlchemyDB(path))
+    experiments = pd.DataFrame(
+        dashboard_data_reader.get_last_experiments(MAX_EXPERIMENTS_NUM)
+    )
+    # TODO: How to filter this column when its values are emoji?
```
Comment (qguyk) [inline entropylab/results/dashboard/dashboard.py:48]: maybe worth disabling the filter until it is solved?

**Review by qguyk — CHANGES_REQUESTED** [review]

File: entropylab/results/dashboard/dashboard.py (hunk 3 above)
Comment (urig) [reply entropylab/results/dashboard/dashboard.py:48]: I'd rather keep the filter as is for this PR and work on a fix for it immediately after in a separate PR if that's alright with you?

File: entropylab/results/dashboard/dashboard_data.py (hunk 2 above)
Comment (urig) [reply entropylab/results/dashboard/dashboard_data.py]: Done. Tx!

File: entropylab/results/dashboard/dashboard.py (hunk 3 above)
Comment (galwiner) [reply entropylab/results/dashboard/dashboard.py:48]: I'm OK with not removing this code on the current PR

**Review by qguyk — DISMISSED** [review]

**Comment by urig** [comment]
@qguyk Cleaned up commits w/ interactive rebase.

**Review by qguyk — APPROVED** [review]

**Pull request merged**